e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts

```diff
@@ -65,7 +65,7 @@ xdescribe('Start Task', () => {
         activitiUser = await identityService.createIdentityUser();
 
         browser.get('/');
-        loginSSOPage.loginSSOIdentityService(browser.params.identityUser.email, browser.params.identityUser.password);
+        loginSSOPage.loginSSOIdentityService(apsUser.email, apsUser.password);
```

Comment:
Do we need to change this? Do we need to create the aps user?

Reply:
We don't really need to create a user, but I guess it's preferred since we delete it at the end of the test.